apps/meteor/app/models/server/models/LivechatInquiry.ts

```diff
@@ -12,17 +15,18 @@ export class LivechatInquiry extends Base {
 		this.tryEnsureIndex({ status: 1 }); // 'ready', 'queued', 'taken'
 		this.tryEnsureIndex({ queueOrder: 1, estimatedWaitingTimeQueue: 1, estimatedServiceTimeAt: 1 });
 		this.tryEnsureIndex({ 'v.token': 1, 'status': 1 }); // visitor token and status
+		this.tryEnsureIndex({ locked: 1, lockedAt: 1 }, { sparse: true }); // locked and lockedAt
```

Comment:
```suggestion
		this.tryEnsureIndex({ locked: 1, lockedAt: 1 }, { sparse: true }); // locked and lockedAt
		this.tryEnsureIndex({ queue: 1 }, { sparse: true }); // queue
```

Adding index on the queue prop as well since it is getting used in query

Reply:
Where's that prop? 👀

Reply:
![image](https://user-images.githubusercontent.com/34130764/171880784-8af808bf-10a5-486a-99b8-45acc40248ff.png)
Here it is

Reply:
Well, that wasn't a new prop but a bug lol good catch there